rollup/core/objects/RollupState__c/RollupState__c.object-meta.xml

```diff
@@ -0,0 +1,172 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Thoughts on adding a few extra goodies for `RollupState__c`? I was thinking...
- Create a custom tab for the object + add the tab to `Rollup.app-meta.xml`
- Add `tabSettings` access in `See_Rollup_App.permissionset-meta.xml`
- Include an `All` list view
  - Maaaaaybe also a 'My Rollup States' list view too? That could be helpful for orgs with multiple admins, and 1 admin wants to easily see just their own records
- Enable reports for the object (`<enableReports>true</enableReports>`)
  - Maaaaaybe enable search too (`<enableSearch>true</enableSearch>)? 

Those options would make it easier for people that want to keep an eye on the generated records + periodically check for orphaned records to delete (if/when needed).

Reply:
All of that makes sense!